### DISCUSSION TITLE
Default N/A是没有默认值的意思吗？

### DISCUSSION THREAD

**Discussion by IsisPolei** [apache/doris#6606] [A - General / Q&A] [answered]
Default  N/A 和Null有什么区别，java在读取该列数据的时候遇到Double类型字段，但是有nan值，怎么才能正确读取？

**Comment by EmmyMiao87** [comment, marked as answer]
Null 和 N/A 不是一个值。
Null 就是 Null，代表这列的值是 Null。
N/A 一般出现在，比如 除0 这种运算的结果就是 N/A
